src/onemod/dtypes/column_spec.py

```diff
@@ -1,18 +1,39 @@
-from typing import List, Type
+from typing import Any, List, Type
```

Comment:
Minor point here as well, we use `List` and `list` for typing, also `Dict` and `dict`. I think we should all switch to `list` and `dict`.

Reply:
Agree with the consistency approach. I did not pay attention to that on this branch but actually it does make decent sense to go through replace instances of this sort of inconsistency here. brb.